scripts/install/opt/seagate/cortx/motr/libexec/m0addb_logrotate.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/sh
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+# script is used to delete the old motr addb stobs
+# argument1: <number of latest addb stob dir to retain>
+# Default number of latest addb stob dir is 2
+# ./m0addb_logrotate.sh 2
+
+usage() { 
+        echo "Usage: bash $(basename $0) [--help|-h]
+                     [-n LogDirCount]
+Retain recent modified directories of given count and remove older motr addb stob directories.
+where:
+-n            number of latest addb stob directories to retain 
```

Comment:
Let's keep two addb directories for both VM and HW, no need to check for get_platform logic.

Reply:
Done